howso/analysis.amlg

```diff
@@ -81,11 +81,21 @@
 			use_case_weights (null)
 			;{type "boolean"}
 			inverse_residuals_as_weights (null)
+			;{type "list" values "list"}
+			shared_deviations_features (null)
```

Comment:
I don't know if this is just for testing or not... but I don't think this form of configuration is gonna fly. I was under the impression we were pretty hard-set on having this configured via feature attributes which would affect the model universally, not on a per-analysis basis. 

But maybe I missed a conversation